app/code/Magento/Review/Api/GetProductRatingSummaryInterface.php

```diff
@@ -0,0 +1,22 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+namespace Magento\Review\Api;
+
+/**
+ * Interface to get product rating summary
+ */
+interface GetProductRatingSummaryInterface
```

Comment:
Is there existing @api class which returns the same data? If so, we need to make sure there is only one @api for the same purpose.